### PR TITLE
feat(inbound-filters): Add health check explanation

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Before:**

image

**After:**

image

Related: https://github.com/getsentry/sentry/issues/54539

### CODE REVIEW

File: src/docs/product/data-management-settings/filtering/index.mdx [hunk 1]
```diff
@@ -16,14 +16,36 @@ These filters are exclusively applied at ingest time and not later in processing
...
-- From certain IP addresses
-- Transactions coming from healthcheck and ping requests
+- **Common browser extension errors**
+- **Events coming from localhost**
+- **Known legacy browsers errors**
+- **Known web crawlers**
+- **By their error message**
+- **From specific release versions of your code**
+- **From certain IP addresses**
+- **Transactions coming from healthcheck and ping requests**
```
[the remark this answers is not in the document]
Comment (vivianyentran) [reply src/docs/product/data-management-settings/filtering/index.mdx:26]: I also see "Filter out hydration errors" in my UI and looks like this is missing from the list.

File: src/docs/product/data-management-settings/filtering/index.mdx [hunk 2]
```diff
@@ -16,14 +16,36 @@ These filters are exclusively applied at ingest time and not later in processing
...
+- `*ready*`
+- `*heartbeat*`
+- `*/health`
+- `*/healthz`
+- `*/ping`
+
+<hr />
+
+A node will be red if it is associated with errors.
+Click "View Full Trace" to display the [Trace View](/product/sentry-basics/tracing/trace-view).
```
Comment (vivianyentran) [inline src/docs/product/data-management-settings/filtering/index.mdx:48]: This feels like a sudden change of topic. It's unclear to me what nodes and traces you're talking about here.

File: src/docs/product/data-management-settings/filtering/index.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply src/docs/product/data-management-settings/filtering/index.mdx:26]: makes sense and good catch. I will update the list

File: src/docs/product/data-management-settings/filtering/index.mdx (hunk 2 above)
Comment (priscilawebdev) [reply src/docs/product/data-management-settings/filtering/index.mdx:48]: I noticed the same but this was already in the docs I don't have the context to change it... I would prefer to leave it there as it is

**Pull request merged**